addon/data/selector-ui.js

```diff
@@ -159,6 +159,72 @@ const ui = (function () { // eslint-disable-line no-unused-vars
     boxBottomEl: null
   };
 
+  /** Displays the box which allows the user to choose between selecting a region,
+      archiving the page, or going to my shots.
+  */
+  exports.SelectMode = {
+    display: function (callbacks) {
+      if (document.querySelector(".pageshot-preview-overlay") !== null) {
+        return;
+      }
+      let oldNode = document.getElementById("pageshot-select-mode-background");
```

Comment:
Given the discussion, I think we should remove these and fix the bug (maybe in a follow up).  This code covers up a bug that I expect will show itself in other ways.